src/index.ts

```diff
@@ -6,6 +6,12 @@
  * This code is licensed under the MIT license (see LICENSE.txt for details).
  */
 
-export * from './currency';
-export * from './format';
-export * from './country';
+import currencies from './currency';
+import countries from './country';
+import format from './format';
+
+export default function (
+  at: string,
+): ReturnType<typeof currencies> & ReturnType<typeof countries> & ReturnType<typeof format> {
```

Comment:
I would import/re-export the *Library interfaces and use those here, instead of the `typeof` syntax